Makefile

```diff
@@ -250,11 +262,11 @@ clean-container:
 clean: clean-container clean-build
 	-$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C ./contrib/packaging/deb clean
 	-$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C ./contrib/packaging/rpm clean
-	-$(QUIET) rm -f GIT_VERSION
-	-$(QUIET) docker builder prune --filter type=exec.cachemount -f
 
-clean-build:
-	-$(QUIET) rm -rf _build
+veryclean:
+	-$(QUIET) $(CONTAINER_ENGINE) image prune -af
+	-$(QUIET) $(CONTAINER_ENGINE) builder prune -af
+	-$(QUIET) $(CONTAINER_ENGINE) system prune -af
```

Comment:
:exploding_head:

From https://docs.docker.com/engine/reference/commandline/system_prune/ :

> $ docker system prune -a --volumes
> 
> WARNING! This will remove:
>         - all stopped containers
>         - all networks not used by at least one container
>         - all volumes not used by at least one container
>         - all images without at least one container associated to them
>         - all build cache
> Are you sure you want to continue? [y/N] y